### PR TITLE
Add integration test for basic functionality

### PR DESCRIPTION
**Pull request by palango**
Adds a basic integration test to cover the basic raiddit functionality. Burning and settlement are still to be added.

### CODE REVIEW

File: raiden/tests/integration/test_raiddit.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    transfer(
+        initiator_app=app0,
+        target_app=app2,
+        token_address=token_address,
+        amount=PaymentAmount(50),
+        identifier=PaymentID(42),
+    )
+
+    gevent.sleep(1)
```
Comment (karlb) [inline raiden/tests/integration/test_raiddit.py:87]: The sleep is ok for a PoC, otherwise I'd complain about potential flakyness, here.

Comment (palango) [reply raiden/tests/integration/test_raiddit.py:87]: Maybe you know if there's a function to wait until the payment has been unlocked completely in all participating nodes? Here node 2 was the problem in some cases.

Comment (karlb) [reply raiden/tests/integration/test_raiddit.py:87]: I don't know any, but I also didn't write many raiden integration tests, so I could easily have missed it.

Comment (palango) [reply raiden/tests/integration/test_raiddit.py:87]: Let's go with this for now then

**Pull request merged**